build/tradable.mjs

```
@@ -1,29 +1,32 @@
   'Warframe Mod',
 ];
+const tradableTypes = [
+  'Arcane',
+  'Captura',
+  'Cut Gem',
+  'Fish',
+  'Focus Lens',
+  'Relic',
+  'Upgrades',
+  ...tradableMods,
+];
```

ESLint

[error] Replace `⏎··'Arcane',⏎··'Captura',⏎··'Cut·Gem',⏎··'Fish',⏎··'Focus·Lens',⏎··'Relic',⏎··'Upgrades',⏎··...tradableMods,⏎` with `'Arcane',·'Captura',·'Cut·Gem',·'Fish',·'Focus·Lens',·'Relic',·'Upgrades',·...tradableMods`

(prettier/prettier)